src/screens/generation-history/ui/generation-history-screen.tsx

```
@@ -0,0 +1,244 @@
+    <div className="flex flex-col gap-8 pb-20 overflow-x-hidden">
+      <div className="sticky top-0 z-20 -mx-6 -mt-6 border-b border-white/5 bg-background-dark/95 px-6 py-6 backdrop-blur-xl sm:px-10">
+        <div className="mx-auto flex w-full max-w-[1600px] flex-col gap-6">
+          <div className="flex flex-col gap-6 md:flex-row md:items-end md:justify-between">
+            <div>
+              <h1 className="text-4xl font-black uppercase leading-tight tracking-[-0.033em] text-white sm:text-5xl">
+                <span className="text-white">Generation</span>{" "}
+                <span className="text-primary">History</span>
+              </h1>
+              <p className="mt-2 flex items-center gap-2 text-xs font-mono tracking-wide text-gray-400">
+                <span className="h-1.5 w-1.5 animate-pulse rounded-full bg-primary" />
+                ALL SYSTEMS OPERATIONAL // ARCHIVE ACCESS
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)